File: fe/src/services/api/routes/dashboard.route.js
import api from "../axios";

export const getAdminStats = async () => {
  const response = await api.get("/dashboard/admin");
  return response.data;
};

export const getUserStats = async () => {
  const response = await api.get("/dashboard/user");
  return response.data;
};

export const getAdminAnalitikWilayah = async () => {
  const response = await api.get("/dashboard/admin/wilayah");
  return response.data;
};

File: be/app/api/routes/dashboard_route.py
"""Dashboard routes"""
from flask import Blueprint
from app.api.controllers import dashboard_controller

dashboard_bp = Blueprint('dashboard', __name__, url_prefix='/api/dashboard')

@dashboard_bp.route('/admin', methods=['GET'])
def admin_stats():
    return dashboard_controller.admin_stats()

@dashboard_bp.route('/user', methods=['GET'])
def user_stats():
    return dashboard_controller.user_stats()

@dashboard_bp.route('/admin/wilayah', methods=['GET'])
def admin_analitik_wilayah():
    return dashboard_controller.admin_analitik_wilayah()